schema.graphql

```diff
@@ -97,3 +97,8 @@ type FundingEvent @entity {
   value: String!
   total_issued_in_funding_round: String!
 }
+
```

Comment:
Don't we need to add external agents stuff to the schema here?

It looks like tooling has stuff for queries on ExternalAgents.

Reply:
It's already in the branch, TickerExternalAgentAction, expand the diff a bit up to line 81

Reply:
It's already in the branch, TickerExternalAgentAction, expand the diff a bit up to line 81

Reply:
My bad, I assumed it would be in this one.